ERROR - 2018-02-24 02:47:01 --> 404 Page Not Found: Faviconico/index
ERROR - 2018-02-24 02:47:46 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240247462253', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:47:46')
ERROR - 2018-02-24 02:49:06 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240249067237', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"2\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:49:06')
ERROR - 2018-02-24 02:49:19 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240249195095', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"4\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:49:19')
ERROR - 2018-02-24 02:49:26 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240249263749', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"4\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:49:26')
ERROR - 2018-02-24 02:49:33 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240249335619', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:49:33')
ERROR - 2018-02-24 02:49:38 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240249380531', '1', 'user-management/privilege_menu', 'Save', '{\\\"menu\\\":\\\"1\\\",\\\"privilege\\\":\\\"1\\\",\\\"access\\\":\\\"2\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:49:38')
ERROR - 2018-02-24 02:50:19 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('20180224025019044', '1', 'user-management/privilege_menu', 'Save', '{\\\"id\\\":\\\"\\\",\\\"name\\\":\\\"dash\\\",\\\"description\\\":\\\"yihaa\\\",\\\"url\\\":\\\"dashboard\\\\/yiha\\\",\\\"icon\\\":\\\"icon-bubbles\\\",\\\"parent\\\":\\\"1\\\",\\\"status\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Saving Data\\\",\\\"message\\\":\\\"Data has been saved.\\\"}', '::1', '1', '2018-02-24 02:50:19')
ERROR - 2018-02-24 02:50:30 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240250304028', '1', 'user-management/privilege_menu', 'Delete', '{\\\"id\\\":\\\"194\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 02:50:30')
ERROR - 2018-02-24 02:53:16 --> Severity: Error --> Call to undefined method Model_privilege_menu::getPrivilege_MenuData() C:\xampp\htdocs\Gmedia\app\controllers\user-management\Privilege_menu.php 155
ERROR - 2018-02-24 03:08:55 --> Severity: Error --> Call to undefined method Model_privilege_menu::updatePrivilege_Menu() C:\xampp\htdocs\Gmedia\app\controllers\user-management\Privilege_menu.php 116
ERROR - 2018-02-24 03:40:01 --> Query error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHEREid'0'ORDER BY SORT DESC LIMIT 1' at line 3 - Invalid query: SELECT level 
		   					Fromcms_menu
		   					WHEREid'0'ORDER BY SORT DESC LIMIT 1
ERROR - 2018-02-24 03:40:01 --> Severity: Error --> Call to a member function num_rows() on a non-object C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 370
ERROR - 2018-02-24 03:44:12 --> 404 Page Not Found: /index
ERROR - 2018-02-24 03:45:45 --> Query error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHEREid='0'ORDER BY sort DESC LIMIT 1' at line 3 - Invalid query: SELECT level 
		   					Fromcms_menu
		   					WHEREid='0'ORDER BY sort DESC LIMIT 1
ERROR - 2018-02-24 03:45:46 --> Severity: Error --> Call to a member function num_rows() on a non-object C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 370
ERROR - 2018-02-24 03:47:19 --> Query error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHEREid='0'ORDER BY sort DESC LIMIT 1' at line 3 - Invalid query: SELECT level 
		   					Fromcms_menu
		   					WHEREid='0'ORDER BY sort DESC LIMIT 1
ERROR - 2018-02-24 03:47:19 --> Severity: Error --> Call to a member function num_rows() on a non-object C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 370
ERROR - 2018-02-24 03:48:02 --> Query error: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'WHEREid='0'ORDER BY sort DESC LIMIT 1' at line 3 - Invalid query: SELECT level 
		   					Fromcms_menu
		   					WHEREid='0'ORDER BY sort DESC LIMIT 1
ERROR - 2018-02-24 03:48:02 --> Severity: Error --> Call to a member function num_rows() on a non-object C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 370
ERROR - 2018-02-24 03:51:39 --> Severity: Notice --> Undefined property: CI_DB_mysqli_driver::$escape_str C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 388
ERROR - 2018-02-24 03:51:42 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240351423144', '1', 'user-management/privilege_menu', 'Update', '{\\\"id\\\":\\\"1\\\",\\\"name\\\":\\\"Dashboarddb\\\",\\\"description\\\":\\\"Manage Dashboarddb\\\",\\\"url\\\":\\\"dashboarddb\\\",\\\"icon\\\":\\\"icon-bar-chart\\\",\\\"parent\\\":\\\"0\\\",\\\"status\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 03:51:42')
ERROR - 2018-02-24 03:52:43 --> Severity: Parsing Error --> syntax error, unexpected ';' C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_privilege_menu.php 388
ERROR - 2018-02-24 03:54:05 --> Severity: Error --> Maximum execution time of 30 seconds exceeded C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 136
ERROR - 2018-02-24 03:56:09 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240356098778', '1', 'user-management/privilege_menu', 'Update', '{\\\"id\\\":\\\"1\\\",\\\"name\\\":\\\"Dashboardioio\\\",\\\"description\\\":\\\"Manage Dashboardioio\\\",\\\"url\\\":\\\"dashboardioio\\\",\\\"icon\\\":\\\"icon-anchor\\\",\\\"parent\\\":\\\"0\\\",\\\"status\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 03:56:09')
ERROR - 2018-02-24 03:57:56 --> Severity: Warning --> mysqli::real_connect(): (HY000/2002): A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 135
ERROR - 2018-02-24 03:57:56 --> Unable to connect to the database
ERROR - 2018-02-24 03:58:06 --> Severity: Error --> Maximum execution time of 30 seconds exceeded C:\xampp\htdocs\Gmedia\sys_ci_3.0\core\Common.php 595
ERROR - 2018-02-24 03:59:44 --> 404 Page Not Found: /index
ERROR - 2018-02-24 04:00:41 --> Severity: Warning --> mysqli::real_connect(): MySQL server has gone away C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 135
ERROR - 2018-02-24 04:00:41 --> Severity: Error --> Maximum execution time of 30 seconds exceeded C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 136
ERROR - 2018-02-24 04:00:41 --> Severity: Warning --> mysqli::real_connect(): (HY000/2006): MySQL server has gone away C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 135
ERROR - 2018-02-24 04:00:41 --> Unable to connect to the database
ERROR - 2018-02-24 04:00:45 --> Severity: Error --> Call to a member function real_escape_string() on a non-object C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 331
ERROR - 2018-02-24 04:02:40 --> Query error: Unknown column 'ip' in 'field list' - Invalid query: INSERT INTO `cms_action_history` (`id`, `id_user`, `url`, `actions`, `data`, `result`, `ip`, `entry_by`, `entry_time`) VALUES ('201802240402409562', '1', 'user-management/employees', 'Update', '{\\\"id\\\":\\\"6\\\",\\\"id_user_type\\\":\\\"1\\\",\\\"id_privilege\\\":\\\"1\\\",\\\"name\\\":\\\"ghazi\\\",\\\"email\\\":\\\"ghazimediazxc\\\",\\\"phone\\\":\\\"8080808089\\\",\\\"address\\\":\\\"holad\\\",\\\"username\\\":\\\"admin\\\",\\\"password\\\":\\\"adminghazi\\\",\\\"status\\\":\\\"1\\\"}', '{\\\"success\\\":true,\\\"title\\\":\\\"Updating Data\\\",\\\"message\\\":\\\"Data has been updated.\\\"}', '::1', '1', '2018-02-24 04:02:40')
ERROR - 2018-02-24 04:02:47 --> 404 Page Not Found: Backend/out
ERROR - 2018-02-24 04:03:32 --> Severity: Error --> Call to undefined method Model_account::loadDbPlatform() C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_account.php 13
ERROR - 2018-02-24 04:03:45 --> 404 Page Not Found: /index
ERROR - 2018-02-24 04:06:21 --> 404 Page Not Found: Parameters/usertype
ERROR - 2018-02-24 04:07:18 --> 404 Page Not Found: parameters/Usertype/index
ERROR - 2018-02-24 04:07:28 --> Severity: Error --> Call to undefined method Model_account::loadDbPlatform() C:\xampp\htdocs\Gmedia\app\models\backend\user-management\Model_account.php 13
ERROR - 2018-02-24 04:59:27 --> Severity: Parsing Error --> syntax error, unexpected ';' C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 43
ERROR - 2018-02-24 04:59:47 --> Severity: Parsing Error --> syntax error, unexpected 'array' (T_ARRAY) C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 46
ERROR - 2018-02-24 05:00:05 --> Severity: Notice --> Undefined property: Usertype::$laod C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 18
ERROR - 2018-02-24 05:00:05 --> Severity: Error --> Call to a member function view() on a non-object C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 18
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined variable: params C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 27
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined index:  C:\xampp\htdocs\Gmedia\app\models\backend\parameters\Model_User_Type.php 68
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined variable: params C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 32
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined index: acccessDelete C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 40
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined variable: params C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 32
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined index: acccessDelete C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 40
ERROR - 2018-02-24 05:00:32 --> Severity: Notice --> Undefined variable: params C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 47
ERROR - 2018-02-24 05:05:43 --> Severity: Notice --> Undefined index: acccessDelete C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 40
ERROR - 2018-02-24 05:05:43 --> Severity: Notice --> Undefined index: acccessDelete C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 40
ERROR - 2018-02-24 05:10:59 --> Severity: Warning --> mysqli::real_connect(): (HY000/2002): A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
 C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 135
ERROR - 2018-02-24 05:10:59 --> Unable to connect to the database
ERROR - 2018-02-24 05:11:19 --> Severity: Error --> Maximum execution time of 30 seconds exceeded C:\xampp\htdocs\Gmedia\sys_ci_3.0\database\drivers\mysqli\mysqli_driver.php 136
ERROR - 2018-02-24 05:17:17 --> 404 Page Not Found: parameters/Usertype/getUsertypeData
ERROR - 2018-02-24 05:19:58 --> 404 Page Not Found: parameters/Usertype/getUsertypeData
ERROR - 2018-02-24 05:20:48 --> Severity: Notice --> Undefined variable: result C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 62
ERROR - 2018-02-24 05:24:09 --> 404 Page Not Found: parameters/Globalparameter/index
ERROR - 2018-02-24 05:32:34 --> 404 Page Not Found: /index
ERROR - 2018-02-24 05:48:09 --> Severity: Parsing Error --> syntax error, unexpected '}' C:\xampp\htdocs\Gmedia\app\controllers\parameters\Usertype.php 77
